ansible_base/feature_flags/urls.py

```diff
@@ -1,12 +1,16 @@
-from django.urls import path
+from django.urls import include, path
 
 from ansible_base.feature_flags import views
 from ansible_base.feature_flags.apps import FeatureFlagsConfig
+from ansible_base.lib.routers import AssociationResourceRouter
 
 app_name = FeatureFlagsConfig.label
 
-api_version_urls = [
-    path('feature_flags_state/', views.FeatureFlagsStateListView.as_view(), name='feature-flags-state-list'),
-]
+router = AssociationResourceRouter()
+
+router.register(r'feature_flags/states', views.FeatureFlagsStatesView, basename='aap_flags_states')
```

Comment:
The serializer for this has an awfully small number of fields. The model has many more fields. Even the management command gives more information.

Were you intending to add some other endpoint which would give the UI the information necessary to render choices and stuff?

Reply:
Hey Alan,
This is intentional for this endpoint.

The endpoint which the UI would use, which contains the remainder of fields, is only exposed via gateway. 
Please see this [view](https://github.com/ansible/aap-gateway/pull/795/files#diff-c636e8a18790306b0cce5de71da895bd085ea6ab342f95f1510f0370c46dbf41R14)

And its corresponding [serializers](https://github.com/ansible/aap-gateway/pull/795/files#diff-c5a86b58f9e7b848abdfdd3dd012793fd9da1b9d62d1900f8c3caaae21161f1fR7)